src/views/Releases/ListReleases/index.jsx

```diff
@@ -64,12 +67,28 @@ function ListPermissions(props) {
     }
   }, [hash, filteredReleases]);
 
+  const handleAccessChange = ({ release, checked }) => {
+    setDialogState({
+      open: true,
+      title: checked ? 'Read/Write?' : 'Read Only?',
```

Comment:
I think this is backwards. When I click the button for CDM-15, which is currently read-write, the title of the dialog is `Read/Write?`